lib/config.js

```diff
@@ -94,7 +94,8 @@ Config.prototype.buildArgs = function () {
     chrome_version_major: chrome_version_parts[0],
     safebrowsing_api_endpoint: this.safeBrowsingApiEndpoint,
     brave_referrals_api_key: this.braveReferralsApiKey,
-    enable_hangout_services_extension: this.enable_hangout_services_extension
+    enable_hangout_services_extension: this.enable_hangout_services_extension,
+    manifest_package: 'com.brave.browser'
```

Comment:
I think you need to add this in the android section because it won't be valid for desktop and will probably cause an error